cmd/templates/hook.tmpl

```diff
@@ -21,7 +21,7 @@ call_lefthook()
   elif bundle exec lefthook -h >/dev/null 2>&1
   then
     bundle exec lefthook $1
-  elif npx lefthook -h >/dev/null 2>&1
+  elif npx @arkweid/lefthook -h >/dev/null 2>&1
   then
     npx lefthook $1
```

Comment:
Should this line be updated too?

Reply:
It may be updated and may be not updated. `npx` seems to search for binaries both in `node_modules` and in system `PATH`, so if `if` statement above if passes then there will be `lefthook` binary in `node_modules`.

Reply:
no, that wont work. maybe both should be checked.  this is for the case when the module is not in the root of the git project. so doing npx lefthook wont run it.

Reply:
Oh yeah, my bad. The line below also needs to be npx @arkweid/lefthook